src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -170,6 +173,16 @@ export class ValueElementRenderer {
     $$(elem).on('blur', () => $$(this.listItem).removeClass('coveo-focused'));
   }
 
+  private toggleAriaHiddenOnFocusBlur(elem: Dom) {
+    elem.on(['focus', 'mouseover'], () => {
+      elem.setAttribute('aria-hidden', 'false');
+    });
+
+    elem.on(['blur', 'mouseout'], () => {
+      elem.setAttribute('aria-hidden', 'true');
+    });
+  }
```

Comment:
I don't see a case where the SVG shouldn't be hidden from the accessibility tree. However, the exclude button probably shouldn't ever be hidden so long as it's visible.